mycity/mycity/utilities/common_types.py

```diff
@@ -0,0 +1,17 @@
+"""
+Common complex types
+
+"""
+
+import typing
+
+StrDict = typing.Dict[str, str]
+
+# This is a work in progress
+# If anything has a very specific type, build it here.
+# It can help us later on with CI linting and strict runtime checking
+# Refer to https://docs.python.org/3/library/typing.html for help
+
+# Use for any parsed JSON, or nested dict structure,
+# until the true structure is known
+ComplexDict = typing.Dict[str, typing.Any]
```

Comment:
Nitpick: If there is any place where this isn't used to represent a parsed JSON? If not, maybe we should rename this to make the association with JSON objects more explicit?

Reply:
That was part of my thoughts for this usage. Ideally I'd like `ComplexDict` to go away entirely, but this was just to get it started.